satellite/metainfo/config.go

```diff
@@ -18,13 +18,26 @@ const (
 	BoltPointerBucket = "pointers"
 )
 
+// RSConfig is a configuration struct that keeps details about default
+// redundancy strategy information
+type RSConfig struct {
+	MaxBufferMem     memory.Size `help:"maximum buffer memory (in bytes) to be allocated for read buffers" default:"4MiB"`
+	ErasureShareSize memory.Size `help:"the size of each new erasure share in bytes" default:"256B"`
+	MinThreshold     int         `help:"the minimum pieces required to recover a segment. k." releaseDefault:"29" devDefault:"4"`
+	RepairThreshold  int         `help:"the minimum safe pieces before a repair is triggered. m." releaseDefault:"35" devDefault:"6"`
+	SuccessThreshold int         `help:"the desired total pieces for a segment. o." releaseDefault:"80" devDefault:"8"`
+	MaxThreshold     int         `help:"the largest amount of pieces to encode to. n." releaseDefault:"130" devDefault:"10"`
+	Validate         bool        `default:"false" help:"toggle flag if rsconfig is enabled"`
+}
+
 // Config is a configuration struct that is everything you need to start a metainfo
 type Config struct {
 	DatabaseURL          string      `help:"the database connection string to use" releaseDefault:"postgres://" devDefault:"bolt://$CONFDIR/pointerdb.db"`
 	MinRemoteSegmentSize memory.Size `default:"1240" help:"minimum remote segment size"`
 	MaxInlineSegmentSize memory.Size `default:"8000" help:"maximum inline segment size"`
 	Overlay              bool        `default:"true" help:"toggle flag if overlay is enabled"`
 	BwExpiration         int         `default:"45"   help:"lifespan of bandwidth agreements in days"`
+	RS                   RSConfig    `help:"reed solomon configurations"`
```

Comment:
e.g. here, we can use `uplink.RSConfig` rather than redefining `RSConfig` in this package

Reply:
Of course we would still need the `Validate` config var somewhere

Reply:
updated